blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java

```diff
@@ -140,12 +137,7 @@ private static LogResource getLogResource(FlowNodeWrapper  node){
     @Override
     public Collection<BlueActionProxy> getActions() {
         // The LogAction is not @ExportedBean but we want to expose its subgraph
-        return ActionProxiesImpl.getActionProxies(node.getNode().getActions(), new Predicate<Action>() {
-            @Override
-            public boolean apply(@Nullable Action input) {
-                return input instanceof LogAction;
-            }
-        }, this);
+        return ActionProxiesImpl.getActionProxies(ImmutableList.of(LogAction.class.getName()), node.getNode().getActions(), this);
```

Comment:
@vivek one exception to this rule here but that will have to wait until we merge Karaoke changes